src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/DocumentHelperTest.java

```diff
@@ -2200,23 +2206,68 @@ void createDocumentTypeItem_createsCorrectly() {
         assertThat(actual.getValue()).isEqualTo(expected);
     }
 
-    @Test
-    void setLegalRepVisibleDocuments_hidesDocuments() {
-        CaseData caseData = new CaseData();
-        DocumentTypeItem et1Doc = DocumentFixtures.getDocumentTypeItem("Visible", "ET1");
-        DocumentTypeItem caseFileDoc = DocumentFixtures.getDocumentTypeItem("Hidden", "Tribunal case file");
-        DocumentTypeItem otherDoc = DocumentFixtures.getDocumentTypeItem("Hidden", "Other");
-        DocumentTypeItem referralsDoc = DocumentFixtures.getDocumentTypeItem("Hidden", "Referral/Judicial direction");
-        DocumentTypeItem nullDoc = DocumentFixtures.getDocumentTypeItem("Visible", null);
+    @Nested
+    class SetLegalRepVisibleDocuments {
+        @Test
+        void hidesDocuments() {
+            CaseData caseData = new CaseData();
+
+            caseData.setDocumentCollection(List.of(
+                getDocumentTypeItem("Visible", "ET1"),
+                getDocumentTypeItem("Hidden", "Tribunal case file"),
+                getDocumentTypeItem("Hidden", "Other"),
+                getDocumentTypeItem("Hidden", "Referral/Judicial direction"),
+                getDocumentTypeItem("Visible", null)));
```

Comment:
minor: move the bracket onto next line to bring it in line with the opening `caseData`